project.godot

```diff
@@ -31,6 +32,7 @@ AreaTitleManager="*res://autoloads/area_title_manager.tscn"
 
 window/size/viewport_height=672
 window/size/resizable=false
+window/size/always_on_top=true
```

Comment:
Let's revert this

Reply:
Must have accidentally left that one while testing...